static/docs.css

```diff
@@ -108,7 +108,7 @@ a.sc-logo img {
   font-size: 16px;
 }
 
-.side-nav .nav-header a, .side-nav .nav-header span {
```

Comment:
Oh right, I see how it works. the inner `span` will inherit the settings from the outer `span.nav-header`.